assets/css/frontend.scss

```diff
@@ -422,63 +422,71 @@ a.sensei-certificate-link {
 		}
 	}
 
-	.sensei-quiz-pagination__actions {
-		display: flex;
-		order: 1;
+	.sensei-quiz-pagination__prev-button {
+		&:before  {
+			@include iconbefore();
+			content: '\f104';
+			margin-right: 0.25em;
+		}
+	}
 
-		@media only screen and (min-width: $tablet-breakpoint) {
-			order: 0;
+	.sensei-quiz-pagination__next-button {
+		&:after  {
+			@include iconafter();
+			content: '\f105';
+			margin-left: 0.25em;
 		}
 	}
+}
 
-	.sensei-quiz-pagination__action {
-		display: inline-flex;
+/**
+ * Quiz Actions
+ */
+#sensei-quiz-actions {
```

Comment:
This needs to be an ID, so the styles are not conflicting with the different themes.